cfgov/jinja2/v1/_includes/molecules/hero.html

```diff
@@ -66,8 +66,9 @@ <h1 class="hero_heading">{{ value.heading | safe}}</h1>
 
         {% if value.image.upload %}
             {% set photo=image(value.image.upload, 'original') %}
-            <div class="hero_image"
-                 style="background-image: url( {{ photo.url }} );"></div>
         {% endif %}
+        <div class="hero_image"
+             style="background-image:
+                    url( {{- photo.url if photo else '' -}} );"></div>
```

Comment:
Should not include the entire `style` attribute if `photo` is not defined, otherwise the background image could be defined with an empty URL. Also should it be `photo.url if photo and photo.url else ''`?